lisa/tools/ethtool.py

```diff
@@ -258,46 +258,62 @@ def _get_device_list(self) -> None:
             if not netdir:
                 continue
             cmd_result = self.node.execute(f"ls {netdir}")
-            if cmd_result.exit_code != 0:
-                raise LisaException(
-                    "Could not find the network device under path {netdir}"
-                    f" exit_code: {cmd_result.exit_code}"
-                    f" stderr: {cmd_result.stderr}"
-                )
-            self._device_list.append(cmd_result.stdout)
+            cmd_result.assert_exit_code(
+                message=f"Could not find the network device under path {netdir}"
```

Comment:
The path will be output by the `assert_exit_code`, so it doesn't need to be repeated.